test/api/application/list_test.go

```diff
@@ -0,0 +1,37 @@
+package application
+
+import (
+	"testing"
+
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+func TestApplicationList(t *testing.T) {
+	samples := Samples()
+	// Create.
+	for i := range samples {
+		Create(t, &samples[i]) // Need modify parent array instead of single record created by the loop (to keep created IDs).
```

Comment:
Updated Create parameter to be the parent array's item to avoid ```range``` making the item copy (which is not a pointer anymore).